src/mame/konami/cobra.cpp

```diff
@@ -55,6 +55,24 @@
         Xilinx XC5204 FPGA
         Konami K001604 (2D tilemaps + 2x ROZ)
         Bt121KPC80 RAMDAC
+
+    TODO:
+    - timings are way off
+     * games run at incorrect speeds, bujutsu runs slow ingame and fast on character select
+     * all games have 15 frames of input lag!
+     * bujutsu: ingame coin counter doesn't decrease when starting a game
+     * bgms have a tendency to skip parts occasionally
+     * occasional performance hits
+    - bujutsu: sometimes crashes
+    - racjamdx: segfaults on boot, error message mentions the 'render_texture_scan' command, CPU core bug?
+    - proper PCI conversions
+     * hook up MPC105 from main and gfx boards (to be added in mpc106.cpp)
+    - hook up TMS57002
+    - figure out TMP47P241 usage and hook up if necessary
```

Comment:
I don't think this can be hooked up now, since it's a 4-bit single-chip MCU with 2 KB of undumped internal OTP.

Reply:
I'm going remove that. On a gfx board pic I have, the MCU is near a two-digit 7-seg LED so it likely drives that. This can easily be simulated and doesn't impact anything in terms of game logic (but quite helpful for diagnosing error codes).